mantle/harness/harness.go

```diff
@@ -240,6 +253,16 @@ func (c *H) setRan() {
 	c.ran = true
 }
 
+func (c *H) NonExclusiveTestStarted() bool {
+	return c.nonExclusiveTestsStarted
+}
+
+func (c *H) SetNonExclusiveTestStarted(testStarted bool) {
+	c.mu.Lock()
+	defer c.mu.Unlock()
+	c.nonExclusiveTestsStarted = testStarted
```

Comment:
I think we should just default to setting `c.nonExclusiveTestsStarted = true` instead of passing a boolean as a parameter. Then we don't have to guard it with a mutex either.